### ISSUE TITLE
[BUG] on_error=1 not working with missing license key

### ISSUE THREAD

**Issue by mastrolube** [dashingsoft/pyarmor#1438]
`pyarmor cfg on_error=1 ` let the full traceback of the **missing license key** be showed

```
D:\Dev\Python\GA\dist>py -3.10 run.py
this license key is expired (1:10909)

D:\Dev\Python\GA\dist>py -3.10 run.py
Traceback (most recent call last):
  File "D:\Dev\Python\GA\dist\run.py", line 1, in <module>
    import GramAddict
  File "D:\Dev\Python\GA\dist\GA\__init__.py", line 2, in <module>
    from .pyarmor_runtime_005291 import __pyarmor__
  File "D:\Dev\Python\GA\dist\GA\pyarmor_runtime_005291\__init__.py", line 2, in <module>
    from .pyarmor_runtime import __pyarmor__
RuntimeError: missing license key to run the script (1:10592)
```

**Comment by jondy** [comment]
Got it, because runtime settings `on_error` is stored in the license key, if no license key, the default value `on_error=0` is used.

I'll check the source and try to find one solution, but if it need change lot of code and make logic complex, it may not be fixed but as one known issue.

**Comment by jondy** [comment]
Fixed in 8.4.0
